netci.groovy

```diff
@@ -80,23 +80,6 @@ commitPullList.each { isPr ->
   }
 }
 
-// Windows Spanish image
```

Comment:
❓ Can we comment this out instead, so when we enable it again we know the update is correct?

Reply:
@sharwell We'll git revert the change either way. How does this help?

Reply:
➡️ Git revert should cover it, thanks!